### PR TITLE
chore: rework AWS private locations installation with JSON task definition

### PR DESCRIPTION
**Pull request by tpetillot**

### CODE REVIEW

File: content/reference/admin/private_locations/aws/installation/images/ecs-service.png
Comment (slandelle) [inline content/reference/admin/private_locations/aws/installation/images/ecs-service.png:1]: Do we need a screenshot of the whole screen, or only the zones of interrest?

File: content/reference/admin/private_locations/aws/installation/index.md [hunk 1]
```diff
@@ -31,177 +31,187 @@ Click Create bucket.
...
-
-Then enter a cluster name (and any custom tags you may want) and click on Create.
+`GatlingControlPlaneEc2Policy` allows the Control Plane to deploy a load generator on EC2.
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "ec2:Describe*",
```
Comment (slandelle) [inline content/reference/admin/private_locations/aws/installation/index.md:76]: Do we need to be more specific?

Comment (tpetillot) [reply content/reference/admin/private_locations/aws/installation/index.md:76]: We could try, I already fine-tune `GatlingControlPlaneConfSidecarPolicy` and `GatlingControlPlaneIAMPolicy`.
But I think we can merge this first.

Comment (slandelle) [reply content/reference/admin/private_locations/aws/installation/index.md:76]: OK

File: content/reference/admin/private_locations/aws/installation/images/ecs-service.png
Comment (tpetillot) [reply content/reference/admin/private_locations/aws/installation/images/ecs-service.png:1]: I'm hesitant, I like breadcrumb being there, and create button to show it's the create page...

Comment (slandelle) [reply content/reference/admin/private_locations/aws/installation/images/ecs-service.png:1]: OK, let's see how it works once published

Comment (slandelle) [reply content/reference/admin/private_locations/aws/installation/images/ecs-service.png:1]: My point is that AWS is going to change this page on a regular basis.

**Review by Isammoc — APPROVED** [review]

**Pull request merged**